src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagEmailService.kt

```diff
@@ -79,37 +79,35 @@ class ProductaanvraagEmailService @Inject constructor(
     ) {
         mailTemplateService.findMailtemplateByName(automaticEmailConfirmation.templateName)?.let {
                 mailTemplate ->
-            val from = MailAdres(
-                email = automaticEmailConfirmation.emailSender.extractEmailAddress(configuratieService),
-                name = null
-            )
-            val to = MailAdres(email = to, name = null)
-            val replyToAddress = automaticEmailConfirmation.emailReply?.let {
-                MailAdres(
-                    email = it.extractEmailAddress(configuratieService),
-                    name = null
-                )
-            }
             val mailGegevens = MailGegevens(
-                from = from,
-                to = to,
-                replyTo = replyToAddress,
+                from = MailAdres(
+                    email = automaticEmailConfirmation.emailSender.extractEmailAddress(configuratieService),
```

Comment:
I was looking at the `AutomaticEmailConfirmation` class (unfortunate that this is still Java but I believe @xiduzo failed to get it to work in Kotlin) and I was wondering if the `emailSender` is even mandatory here? I would assume so but from the class it is not clear. I would expect a `@NotNull` on `emailSender` if it was? It is isn't we should probably do a null check here `automaticEmailConfirmation.emailSender?.let { ..} ?: { .. }`?

Reply:
It's possible to get null via REST and also model allows null value. Will see if I can make this mandatory